---
layout: minimal
authors:
  - "[cbenoit](www.linkedin.com/in/clement-benoit)"
date: 2024-02-15
---

# A quick overview of GSEA analysis

## Why GSEA Analysis ?

Genome wide expression analysis has become become a mainstay of genomics research. However, there is still a wide range of tools for interpreting these gene expression profiles. 
They all have pros, cons, and are still evolving. This, coupled with the fact that these studies rely on the testing of a large number of hypotheses and relatively small sample sizes, lead that whole-genome expression studies in particular, to be often not reproducible. 
This is why reproducibility is one of the major challenges facing studies involving whole-genome expression data. [^1], [^2], [^3]

Finally, interpreting lists of thousands of differentially expressed genes is a tedious exercise for the biologist. 

The GSEA, by dezooming from the scale of the gene to that of the pathway. Improves the reproducibility of studies, while facilitating their interpetation.

## Principles

### What is the question ?

Let's says you have ranked a gene list <b><i>L</i></b> according to your favourite metric. The GSEA tries to answer the following question : <br/>
<p className="popacity">
Given a gene set <b><i>S</i></b> : Does the genes belonging to <b><i>S</i></b> tends to occur toward the top (or the bottom) of the list <b><i>L</i></b>, 
in which case the gene set is correlated with the phenotypic class distinction.
</p>
Of course we will do as many independant tests as we have genes sets to try.
A [multiple testing correction](https://www.firalis.com/products/fimics-cardiac-ruo-kit-panel) should then be considered.

### The Methods

### To go further

There exists a variant of GSEA called FGSEA for <u>F</u>ast <u>G</u>ene <u>S</u>et <u>E</u>nrichment <u>A</u>nalysis.<br/>

Another common approach to perform pathways analysis is the [Gene Ontology Enrichment analysis](https://geneontology.org/docs/go-enrichment-analysis/).

[^1]:  Timothy E. Sweeney , Winston A. Haynes , Francesco Vallania , John P. Ioannidis
and Purvesh Khatri. (2017). *Methods to increase reproducibility in differential gene expression via meta-analysis*. **Nucleic Acids Research**, Volume 45(Issue 1), Page Range. [DOI](https://doi.org/10.1093/nar/gkw797)
[^2]: Steven N Goodman , Daniele Fanelli , John P A Ioannidis. (2016). *What does research reproducibility mean? *. **Sci Transl Med**, 8(341),12. [DOI](https://doi.org/10.1126/scitranslmed.aaf5027)
[^3]: Lianbo Yu. (2020). *RNA-Seq Reproducibility Assessment of the Sequencing Quality Control Project*. **Cancer Inform**, Volume 19 [DOI](https://doi.org/10.1177/1176935120922498)
{/* [^1]: Doe, J., & Smith, A. (Year). *Title of the Publication*. **Journal Name**, Volume(Issue), Page Range. [DOI](https://doi.org/xxxx) */}